.gitignore

```diff
@@ -172,3 +186,9 @@ nbdist/
 nbactions.xml
 nb-configuration.xml
 .nb-gradle/
+
+### Eclipse ###
```

Comment:
I agree but we chose to add many editors already (phpstorm, sublime text, atom) in order to avoid people committing them by mistake.
We also added all temp from most OS.